src/mvBasicWidgets.cpp

```diff
@@ -6271,23 +6303,19 @@ DearPyGui::draw_filter_set(ImDrawList* drawlist, mvAppItem& item, mvFilterSetCon
 	if (item.config.width != 0)
 		ImGui::PushItemWidth((float)item.config.width);
 
-	if (config.imguiFilter.IsActive())
-	{
+	if (config.imguiFilter.IsActive()) {
```

Comment:
The old placement of curly braces was more consistent; besides, such changes bog up `git blame` for no reason (same for empty lines below)...